src/bricks/transformers/temporaryInfo/popoverUtils.ts

```diff
@@ -138,15 +153,17 @@ function reclaimTooltip(tooltip: HTMLElement): void {
 
   // Set popper to null before destroying to avoid deleting the tooltip
   const popper = popperMap.get(tooltip);
-  // https://github.com/floating-ui/floating-ui/issues/538
-  popper.state.elements.popper = null;
-  popper.destroy();
-  popperMap.delete(tooltip);
+  if (popper) {
+    // https://github.com/floating-ui/floating-ui/issues/538
+    popper.state.elements.popper = null as unknown as HTMLElement;
```

Comment:
Untested, but the linked issue seems to suggest that this works too:

```suggestion
    popper.state.elements.popper = document.documentElement;
```

I'll test it

Reply:
@fregante did you ever test this?